examples/landmarks/at.html

```diff
@@ -281,4 +281,4 @@ <h2 id="id3">Related W3C Documents</h2>
     </script>
     <script type="text/javascript" src="js/SkipTo.min.js"></script>      
   </body>
-</html>
```

Comment:
Yup, that character they use is pretty odd. I resisted commenting the same thing on your last PR 😝